packages/web-components/src/wallet-connection/watchWallet.ts

```diff
@@ -100,6 +82,7 @@ export const watchWallet = (
       smartWalletStatusNotifierKit.updater.updateState(
         harden({ provisioned: true }),
       );
+      // @ts-expect-error xxx
```

Comment:
It might be possible to provide a type to `chainStorageWatcher.watchLatest` to resolve this